logstash-core/src/main/java/org/logstash/instrument/witness/QueueWitness.java

```diff
@@ -49,10 +95,9 @@ public void type(String type) {
 
         /**
          * Gets the type of queue
-         *
          * @return the queue type.
          */
-        public String type() {
+        public String type(){
```

Comment:
Not sure what happened there... my formatter has the space defined. I will re-run the formatter.